en/docs/administer/configuring-cache-layers.md

```diff
@@ -0,0 +1,168 @@
+# Configuring Cache Layers
+
+WSO2 Identity Server has multiple cache layers which are used to improve performance. The following configurations that can be configured in the `deployment.toml` file in the `<IS_HOME>/repository/conf/` directory are used to manage and configure the cache layers.
+
+```toml tab="Sample identity application management cache layer"
+[cache.framework_session_context_cache]
+enable = "true"
```

Comment:
Set  all boolean value without quotation marks/speech marks
e.g. enable = true
please do the same in configs  below also